ntbs-service/Services/NotificationChangesService.cs

```diff
@@ -207,6 +208,11 @@ private IEnumerable<AuditLog> GetCanonicalLogs(List<AuditLog> group)
                 return new[] { log };
             }
 
+            if (group.Count == 1 && group.Single().EventType == AuditEventType.REJECT_POTENTIAL)
+            {
+                return new List<AuditLog>();
```

Comment:
Probably worth adding a comment here saying that we don't want rejecting a potential match to appear in the notification changes